app/controllers/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
```

Comment:
bỏ thêm code xử lý khi nil nhé

`return if @user`
`flash[:warning] = "User not found"`
`redirect_to root_path`